src/utils/dif_upload.rs

```diff
@@ -1345,7 +1345,7 @@ fn poll_dif_assemble(
                     k => format!(" {:#}", k),
                 },
                 ParsedDif::BcSymbolMap(_) => String::from("bcsymbolmap"),
-                ParsedDif::PList(_) => String::from("plist"),
+                ParsedDif::UuidMap(_) => String::from("plist"),
```

Comment:
I think this only prints on the console, right? not sure if we should also change that string

Reply:
😱  yes, you're entirely right, updated.